features/NewCustomerRegistration.feature

```diff
@@ -0,0 +1,29 @@
+Feature: New customer Registration
+
+
+Background: customer Registration
+	Given the user navigates to Homepage
+	When the user clicks on the create a account button
```

Comment:
@karthikeyan7010 Background is used to set the pre-conditions or context not test scenario